app/views/physical_ticket/show.pdf.prawn

```diff
@@ -0,0 +1,62 @@
+prawn_document(filename: @file_name, page_layout: :portrait, :page_size =>'A4' ) do |pdf|
+  # Vertical Layout
+  top = pdf.bounds.top
+  bottom = pdf.bounds.bottom
+  left = pdf.bounds.left
+  right = pdf.bounds.right
+  mid_vertical = (pdf.bounds.top-pdf.bounds.bottom)/2
+  mid_horizontal = (pdf.bounds.right-pdf.bounds.left)/2
+  x = 0
+
+  pdf.move_down mid_vertical
+  pdf.dash(2, :space => 1)
+  pdf.stroke_horizontal_rule
+  pdf.stroke_vertical_line pdf.bounds.top, pdf.bounds.bottom, :at => mid_horizontal
+  pdf.move_up mid_vertical
+  pdf.draw_text "TICKET HOLDER", :at => [x,pdf.cursor-30], :size => 17
+  pdf.dash(2, :space => 0)
+  pdf.stroke_rectangle  [x, pdf.cursor-50], 230, 150
+  pdf.move_down 80
+  pdf.draw_text "NAME", :at => [x+10,pdf.cursor], :size => 13
+  pdf.fill_color "808080"
+  pdf.draw_text "#{@user.name}", :at => [x+10,pdf.cursor-25], size: 20
+  pdf.fill_color "000000"
+  pdf.draw_text "EMAIL", :at => [x+10,pdf.cursor-50], :size => 13
+  pdf.fill_color "808080"
+  pdf.draw_text "#{@user.email}", :at => [x+10,pdf.cursor-75], size: 20
+  pdf.fill_color "000000"
+  pdf.move_up 20
+  if @conference.picture?
+    if @conference.picture.width > @conference.picture.height
```

Comment:
I see a problem here. If the image is 120x119 (I am writing `widht x height`, not sure if that is correct :rofl:) you will print the image in 120x119, although it is supposed that your maximum height is 70. If it is right that our maximum height is 70 and our maximum width 120, then the condicion should look like:

``` Ruby
if 7 * @conference.picture.width > 12 * @conference.picture.height
```

With this condition:

120x119 -> 71x70
120x70 -> 120x70
1200x600 -> 120x60
600x700 -> 60x70

and so on